Mathlib/Data/DFinsupp/Defs.lean

```diff
@@ -1142,6 +1218,29 @@ def equivCongrLeft [∀ i, Zero (β i)] (h : ι ≃ κ) : (Π₀ i, β i) ≃ Π
     rw [comapDomain'_apply, mapRange_apply, comapDomain'_apply, Equiv.cast_eq_iff_heq,
       h.apply_symm_apply]
 
+@[simp]
+theorem equivCongrLeft_refl [∀ i, Zero (β i)] :
```

Comment:
in the section above, you have `variable ... [∀ i, Zero (β i)]`. Can you do the same here?